draganddrop/src/main/res/drawable/drag_and_drop_ic_text_fields_black.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
```

Comment:
I think you can also add the drag and drop app icon from the bundle app! https://github.com/microsoft/surface-duo-sdk-samples-kotlin/blob/main/SurfaceDuoSamples/src/main/res/drawable/thumbnail_drag_and_drop.xml